libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -41,10 +43,24 @@ const AP_Param::GroupInfo AP_Camera::var_info[] = {
     // @Path: AP_Camera_Params.cpp
     AP_SUBGROUPINFO(_params[1], "2", 13, AP_Camera, AP_Camera_Params),
 #endif
+#if AP_CAMERA_RUNCAM_ENABLED
+    // @Group: 1_RC_
+    // @Path: AP_RunCam.cpp
+    AP_SUBGROUPVARPTR(_backends[0], "1_RC_", 14, AP_Camera, _backend_var_info[0]),
```

Comment:
Is this how we normally handle extra parameters required by a particular backend?  I wonder if @peterbarker has advice on this

Reply:
Yeah, this is pretty typical of the way we do this.